tests/conftest.py

```diff
@@ -98,6 +98,16 @@ def session_factory(mock_config):
 def mock_config(mocker):
     config = Config(aleph.config.get_defaults())
 
+    config_file = Path.cwd() / "config.yml"
+
+    if config_file.exists():
+        with config_file.open() as f:
+            user_config = f.read()
+
+        # Little trick to allow empty config files
+        if user_config:
+            config.yaml.loads(user_config)
```

Comment:
Would it be worth renaming one of these variables for typing coherence ?
( `user_config_raw: str` and `user_config: dict` for example)

Reply:
yeah that would make sens, let's do it

Reply:
It should be better now, I took the occasion to fix a bug because cli_args was returning a string a not a Path object